uber/configspec.ini

```diff
@@ -559,6 +541,18 @@ name = string(default="")
 path = string(default="")
 
 
```

Comment:
I think the comments explain this, and the default config seems sensible, but let me know if anyone has any questions.